pallets/handles/README.md

```diff
@@ -0,0 +1,74 @@
+For example, while the display may have diacriticals, the handle is stored without diacriticals.
+So `Zoë` and `Zoe` would resolve to the same handle.
+
+### Homoglyph Attack Resistence
+
+Two or more characters appear the same to the user are [homoglyphs](https://en.wikipedia.org/wiki/Homoglyph).
+To prevent most homoglyph attachs where one user attempts to impersonate another, the user's Base Handle is converted to a canonical version of the handle.
+The canonical version determines the suffix series that is chosen.
+Thus `alice` and `a1ice` (with a one instead of an `L`) can never have the same suffix.
+
+
+### Actions
+
+The Handles pallet provides for:
+
+- Unique identifier for an MSA
+- Creation by proxy
+- Tokenless handle removal
+- Shuffled sequences for Suffixes
+
+## Interactions
+
+### Extrinsics
+
+| Name/Description                 | Caller        | Payment | Key Events                                                                                                    | Runtime Added |
+| -------------------------------- | ------------- | ------- | ------------------------------------------------------------------------------------------------------------- | ------------- |
+| `claim_handle`<br />Claim a handle with the given  | Provider or MSA Owner | Capacity or Tokens  | [`HandleClaimed`](https://frequency-chain.github.io/frequency/pallet_handles/pallet/enum.Event.html#variant.HandleClaimed) | 27             |
+| `retire_handle`<br />Retiren a handle. Retired handles + suffix are never reused.   | MSA Owner | Free  | [`HandleRetired`](https://frequency-chain.github.io/frequency/pallet_handles/pallet/enum.Event.html#variant.HandleRetired) | 27             |
+| `change_handle`<br />Convenous method to retire and then claim a new handle  | Provider or MSA Owner | Capacity or Tokens  | [`HandleRetired`](https://frequency-chain.github.io/frequency/pallet_handles/pallet/enum.Event.html#variant.HandleRetired), [`HandleClaimed`](https://frequency-chain.github.io/frequency/{pallet_name}/pallet/enum.Event.html#variant.HandleClaimed) | 47             |
+
+See [Rust Docs](https://frequency-chain.github.io/frequency/pallet_handles/pallet/struct.Pallet.html) for more details.
+
+### State Queries
+
+Note: RPC use is suggested over the direct state queries for handles.
+
+| Name      | Description         | Query                    | Runtime Added |
+| --------- | ------------------- | ------------------------ | ------------- |
+| Get Handle by MSA Id  | Returns the Display Handle and the block it was claimed   | `msaIdToDisplayName` | 29             |
+| Get MSA Id by canonical string and suffix  | Uses the stored canonical lookup string NOT the display handle   | `canonicalBaseHandleAndSuffixToMSAId` | 29             |
+
+See the [Rust Docs](https://frequency-chain.github.io/frequency/pallet_handles/pallet/storage_types/index.html) for additional state queries and details.
```

Comment:
Yes, I left out the canonical to suffix index. It just doesn't make sense for anyone to really use it directly, but that's why we always have this line.